.golangci.yaml

```diff
@@ -12,4 +12,4 @@ linters:
     - unused
 issues:
   max-issues-per-linter: 0
-  max-same-issues: 0
\ No newline at end of file
+  max-same-issues: 0
```

Comment:
This whitespace change in the linter config is tagging DRE for review; if you revert this change then you'll only need review/approval from @d-bhola.

Reply:
Oh wait, I'm wrong; the changes to `config.go` are technically shared, so that requires separate sign-off either way; taking care of that now.